File: msgraph-mail/go/utilities/users/mailFolders/messages/singleValueExtendedProperties/single_value_extended_properties_response.go
package singlevalueextendedproperties
import (
    "github.com/microsoft/kiota-samples/msgraph-mail/go/utilities/users"
    "github.com/microsoft/kiota/abstractions/go/serialization"
)
type SingleValueExtendedPropertiesResponse struct {
    additionalData Map<String, Object>;
    nextLink *string;
    value []SingleValueLegacyExtendedProperty;
}

File: msgraph-mail/swiftconsoleapp/Sources/swiftconsoleapp/Users/Item/Messages/Item/SingleValueExtendedProperties/SingleValueExtendedPropertiesResponse.swift
extension Swiftconsoleapp.Users.Item.Messages.Item.SingleValueExtendedProperties {
    public class SingleValueExtendedPropertiesResponse : IAdditionalDataHolder {
        public var additionalData: IDictionary<string, object>
        public var nextLink: String
        public var value: [SingleValueLegacyExtendedProperty]
    }
}
